docs/getting_started.md

```diff
@@ -635,19 +641,36 @@ Edit the `roles/deprovision-my-pg-apb/tasks/main.yml` and delete the created res
 At this point, the APB can create a fully functional Postgres database to our cluster.  Let's test it out now.
 
 ##### Bind - Execute from the user interface
-To test your app, we'll bind a hello-world app to the provisioned Postgres database.  You can use the application previously created in the **_provision_** section of this tutorial or you can use the [hello-world-apb](https://github.com/fusor/apb-examples/tree/master/hello-world-apb) from apb-examples. Be sure to replace the `ansibleplaybookbundle` org with your own and push to your broker or your own dockerhub repository.  Provision both `my-test-apb` and `my-pg-apb` to a project.
+To test your app, we'll bind a hello-world app to the provisioned Postgres database.  You can use the application previously created in the **_provision_** section of this tutorial or you can use the [hello-world-apb](https://github.com/ansibleplaybookbundle/hello-world-apb). Provision the `my-test-apb` project first.  Then, we'll want to provision `my-pg-apb` and select the option to `Create a secret...`.
```

Comment:
Provision ~~the~~ `my-test-apb` ~~project~~ first

Reply:
fixed